VSTSRestApiSamples.UnitTests/GettingStarted/AuthenticationTest.cs

```diff
@@ -22,14 +22,44 @@ public void TestCleanup()
             _configuration = null;
         }
 
-        [TestMethod, TestCategory("REST API")]
-        public void GettingStarted_Authentication_PersonalAccessToken_Success()
+        [TestMethod, TestCategory("REST API - Authentication")]
+        public void GettingStarted_InteractiveADAL_Success()
         {
             // arrange
             Authentication request = new Authentication();
 
             // act
-            var response = request.PersonalAccessToken(_configuration.UriString, _configuration.PersonalAccessToken);
+            var response = request.InteractiveADAL(_configuration.AccountName, _configuration.ApplicationId);
```

Comment:
Correct.  Unfortunately this is the only way the current tests "run".  I'm going to also ship a standalone sample.